src/main/java/com/dataloom/graph/core/LoomGraph.java

```diff
@@ -9,25 +9,35 @@
 import com.dataloom.graph.core.objects.GraphWrappedVertexId;
 import com.dataloom.graph.core.objects.LoomEdge;
 import com.dataloom.graph.core.objects.LoomVertex;
+import com.dataloom.graph.core.objects.VertexLabel;
 import com.dataloom.hazelcast.HazelcastMap;
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
 
 public class LoomGraph implements LoomGraphApi {
     
+    private static UUID DEFAULT_GRAPH_ID = new UUID(0, 0);
     private UUID graphId;
     
-    private IMap<GraphWrappedVertexId, LoomVertex> vertices;
+    private static IMap<GraphWrappedVertexId, LoomVertex> vertices;
```

Comment:
Why are you changing `vertices` and `verticesLookup` to being static, and changing the constructor to a static `init` method?